tests/drivers/uart/uart_async_api/src/test_uart.h

```diff
@@ -31,7 +31,8 @@
 #define UART_DEVICE_NAME DT_LABEL(DT_NODELABEL(sercom1))
 #elif defined(CONFIG_BOARD_NUCLEO_F103RB) || \
 	defined(CONFIG_BOARD_NUCLEO_G071RB) || \
-	defined(CONFIG_BOARD_NUCLEO_G474RE)
+	defined(CONFIG_BOARD_NUCLEO_G474RE) || \
```

Comment:
Pure comment on  that part:
Isn't is possible to factorize a bit here and use `DT_NODELABEL(arduino_serial)` on a number of boards ?

Reply:
sort the uart instance in the `test_uart.h` depending on the stm32 target board

Reply:
ok, that's a bit better.